packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -39,7 +40,7 @@ const SettingsSidebar = () => {
 		// should contain the sidebar that will be active when the toggle button is pressed. If a block
 		// is selected, that should be edit-post/block otherwise it's edit-post/document.
 		let sidebar = select( 'core/interface' ).getActiveComplementaryArea(
-			'core/edit-post'
+			editPostStore
```

Comment:
I think that this line might be one of the culprits for some failing tests. Forgot the `.name` in this case.